### PR TITLE
PXC-2967: Pxc rel 5.6.47

### PR DESCRIPTION
**Pull request by dutow**

### CODE REVIEW

File: gcs/src/gcs_group.cpp [hunk 1]
```diff
@@ -816,23 +816,19 @@ gcs_group_handle_join_msg  (gcs_group_t* group, const gcs_recv_msg_t* msg)
...
                 }
             }
             else {
-                if (GCS_NODE_STATE_JOINED == sender->status) {
-                    gu_info ("%d.%d (%s): State transfer %s %d.%d (%s) complete.",
-                             sender_idx, sender->segment, sender->name, st_dir,
-                             peer_idx, peer ? peer->segment : -1, peer_name);
-                }
-                else {
+                if (GCS_NODE_STATE_JOINED != sender->status) {
```
Comment (kennt-percona) [inline gcs/src/gcs_group.cpp]: Do we need this if statement?  isn't the else already in this condition?

Comment (dutow) [reply gcs/src/gcs_group.cpp]: Good catch, removed.

**Comment by kennt-percona** [comment]
The GALERA-REVISION file needs to be updated with the commit hash of the galera tag.

**Comment by dutow** [comment]
@kennt-percona Done

**Review by kennt-percona — APPROVED** [review]
lgtm

**Pull request merged**